Duplicati/Library/Main/Operation/BackupHandler.cs

```diff
@@ -503,9 +506,6 @@ private async Task RunAsync(string[] sources, Library.Utility.IFilter filter)
                         // Make sure we have the database up-to-date
                         await db.CommitTransactionAsync("CommitAfterUpload", false);
 
-                        // TODO: Remove this later
-                        m_transaction = m_database.BeginTransaction();

```

Comment:
It's not clear to me why the `TODO` comment about removing this exists, so I left it in above.

Reply:
I think the reason for the `TODO` was that I working on rewriting all the operations to use channels and a single (shared) database transaction. But for time constraint reasons, I had to stop that and just implement the backup/upload part with channels and leave the other operations as they were. They require a transaction to be active to avoid writing partial results, which is why the transaction is started.